test/specs/testCase4.js

```diff
@@ -0,0 +1,18 @@
+import { assert } from 'chai';
+
+describe('Test case 4 ', () => {
+    it('Check login', async () => {
+        await browser.url(`https://www.saucedemo.com/`) 
+        const usernameField = await $('#user-name');
+        await usernameField.addValue('performance_glitch_user');
+        const passwordField = await $('#password');
+        await passwordField.addValue('secret_sauce');
+        const loginButton = await $('#login-button');
+        await loginButton.click();
+        
+        await browser.waitUntil(async () =>{
+            const shopingCartContainer = await $('#shopping_cart_container');
+            return shopingCartContainer.isDisplayed();
+        });     
+    })
```

Comment:
required assertion is missing, also locator #shopping_cart_container is not unique across the app, so you cannot be 100% sure that after login you are landed on products page
![image](https://github.com/valkangeorgiev/vention-tasks/assets/98515302/b2a0131c-f0b4-4b2a-963a-ec8b7a11a281)